app/lib/classification-queue.js

```diff
@@ -40,43 +43,35 @@ class ClassificationQueue {
   }
 
   flushToBackend() {
-    const queue = this._loadQueue();
-
-    if (queue.length > 0) {
-      console.log('Saving queued classifications:', queue.length);
-      queue.forEach((classificationData) => {
-        this.apiClient.type('classifications').create(classificationData).save().then((actualClassification) => {
+    const pendingClassifications = this._loadQueue();
+    const failedClassifications = [];
+    this._saveQueue(failedClassifications);
+
+    if (pendingClassifications.length > 0) {
```

Comment:
This if can be removed, since the promise resolves with an empty array when the map is empty.